datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -3049,17 +3051,54 @@ pub enum Partitioning {
     DistributeBy(Vec<Expr>),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum ColumnUnnestType {
+    List(Vec<ColumnUnnestList>),
```

Comment:
Can you please add documentation to `List` about what it is (ideally with an example of when it is used and how it is different to `Inferred`?

Reply:
comment addressed